deployment/ansible/roles/cac-tripplanner.database/vars/main.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I think that it would be better to surface this in `group_vars/development_template` vs. `vars` so that they are more visible. The `postgres_*` ones in `all` possibly, since they are needed to provision the `app` virtual machine.

Reply:
@hectcastro Does ansible interact with any of the production resources?

Reply:
Yeah, Ansible applies `app.yml` and `otp.yml` to EC2 instances via Packer as part of the AMI creation process: https://github.com/azavea/cac-tripplanner/blob/develop/deployment/packer/cac.json#L104-L125

But the database setup is all RDS, so `cac-tripplanner.database` won't get used in that context.

Reply:
Since before the upgrade, the `postgresql_*` variables were only scoped to `cac-tripplanner.database`, I don't have a strong desire to expand the scope. 

Originally, the `postgresql_*` variables were in `cac-tripplanner.database/defaults/main.yml`.

After including the `azavea.postgresql` and `azavea.postgis` modules, I moved them to `cac-tripplanner.database/vars/main.yml`.

This is what we do in HunchLab:

https://github.com/hunchlab/hunchlab/blob/develop/deployment/ansible/roles/hunchlab.database/vars/main.yml.

As an alternative solution, maybe we could define the variables in the `database.yml` playbook?

I experimented with surfacing these variables in `group_vars/development_template` and there was a conflict with an existing `postgis_version` variable.

Reply:
I missed what you were referring to in the `cac-tripplanner.app/defaults/main.yml` file. 

Maybe as a compromise, we could expand the scope for `postgres_*`, but not `postgresql_*` and `postgis_*`?

Reply:
After sleeping on it, I think an approach of putting `postgres_*` in `all` and `postgresql_*` in `development` works.

See this example from Cicero: 

https://github.com/azavea/cicero/blob/774967531d832e8665906ba8520a623fb1263c22/deployment/ansible/group_vars/all#L26

There's still the issue of the conflicting `postgis_version` var. It looks like this stuff is supposed to be written to `/etc/cac_secrets`, where it's then parsed into an object in a Python script. But, I can't find any code that actually saves it, unless this is an Ansible built in.

Reply:
I like the approach from Cicero (and also MMW) because it has historically made it easier for folks who aren't worried about provisioning details day-to-day see the versions of dependencies at a glance.

The conflict with `postgis_version` is likely something that we should try to resolve explicitly. It might require renaming the variable that takes the tuple vs. the one used by the role. I can't take a closer to provide a better response right now, but I can take a look a little later to see if there is a good way to resolve the conflict.